ddtrace/tracer.py

```diff
@@ -285,6 +285,9 @@ def trace(self, name, service=None, resource=None, span_type=None):
             span_type=span_type,
         )
 
+    def current_root_span(self):
```

Comment:
I'd say to write a docstring here with an example to explain how to retrieve the root span